factory-reset/eos-factory-reset-users

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+ADM_GROUP="sudo"
+
+if [ -f /etc/adduser.conf ]; then
+    . /etc/adduser.conf
+fi
+
+first_uid=${FIRST_UID:-1000}
```

Comment:
It would be interesting to know if this value has ever changed over the history of Endless. If it was increased then it would mean some old user accounts would potentially not be deleted by this script.

Same consideration for any future changes to this number, although we could adjust this script at the same time (if we remember this detail).

Reply:
Unless ostree can provide any sort of history for `/etc/adduser.conf` I don't know of any quick way to do that. For future improvements of the factory reset procedure we should follow @dbnicholson's suggestion and restore `/usr/etc/passwd`, but this would require other considerations like making sure none of the daemons that have system users are running, and perhaps cleaning-up `/var`.

Reply:
You can check the entire package history on OBS.

Reply:
We don't have history before EOS 2.6 or between 3.0 and 3.2 on OBS, so I ended up checking the versions on the packages.txt files for every release. None of these is what I would call quick though, thus my previous question.

Anyway, `adduser.conf` was the same between EOS 2.0 and EOS 3.4, when `LAST_UID` and `LAST_GID` where increased from 29999 to 59999, but `FIRST_UID` was kept the same, so we should be fine. I could not find anything about EOS < 2.0.